docker-compose.yaml

```diff
@@ -113,6 +114,13 @@ services:
       - BUILDTESTS=OFF
     command: bash -c "./scripts/build-android.sh"
 
+  cordova:
+    extends: all
+    environment:
+      - BUILDSWIGJAVA=ON
+      - BUILDCORDOVA=ON
+    command: bash -c "npm install -g upm-cordova-binding && ./scripts/run-cmake.sh && cd build && make -j8 && make -j8"
```

Comment:
We should add this to the Docker Image! instead of running an installation each time you generate the bindings, the upm-cordova-bindings will be installed already in the upm-all image that CI downloads. 

Also with this you depend on a working internet connection to run the cordova bindings generation (which is not always possible for developers in other countries  )

Reply:
Done.